app/assets/javascripts/discourse/tests/acceptance/bootstrap-mode-notice-test.js

```diff
@@ -34,5 +35,12 @@ acceptance("Bootstrap Mode Notice", function (needs) {
       "/wizard/steps/hello-world",
       "it transitions to the wizard page"
     );
+
+    set(this.siteSettings, "bootstrap_mode_enabled", false);
+    await settled();
+    assert.ok(
+      !exists(".bootstrap-mode-notice"),
+      "removes the bootstrap mode notice when bootstrap mode is disabled"
```

Comment:
We could be less verbose (sorry 😅)

```suggestion
      "removes the notice when bootstrap mode is disabled"
```